packages/renderer/src/lib/container/container-utils.ts

```diff
@@ -38,6 +38,9 @@ export class ContainerUtils {
         return `${composeService}-${composeContainerNumber}`;
       }
     }
+    if (containerInfo.Names.length === 0) {
+      return '';
+    }
     return containerInfo.Names[0].replace(/^\//, '');
```

Comment:
could you add a test in container-utils.spec.ts so we're sure we have a regression test